nimare/tests/test_meta_kernel.py

```diff
@@ -179,7 +179,7 @@ def test_mkdakernel_1mm(testdata_cbma):
     Test on 1mm template.
     """
     id_ = "pain_01.nidm-1"
-    kern = kernel.MKDAKernel(r=4, value=1)
+    kern = kernel.MKDAKernel(r=4, value=1, low_memory=True)
```

Comment:
since the code run for `low_memory=True` is diverging from `low_memory=False`, it would probably be a good idea to test the difference between the two, with something like the following:

```
@pytest.mark.parametrize(
    "kern, kwargs",
    [
        (kernel.ALEKernel, {"sample_size": 20}),
        (kernel.KDAKernel, {}),
        (kernel.MKDAKernel, {}),
    ],
)
def test_kernel_low_high_memory(testdata_cbma, tmp_path_factory, kern, kwargs):
    """Test low_memory kernel setting"""
    kern_low_mem = kern(low_memory=True, **kwargs)
    kern_high_mem = kern(low_memory=False, **kwargs)
    trans_kwargs = {"dataset": testdata_cbma, "return_type": "array"}
    assert np.all(
        kern_low_mem.transform(**trans_kwargs) == kern_high_mem.transform(**trans_kwargs)
    )
```

So in case one of the methods changes in the future, we can be sure they give equivalent outputs.

Reply:
Great thinking! Thanks!